data/beta/field/datatype.yaml

```diff
@@ -3,4 +3,4 @@ datatype: string
 field: datatype
 phase: beta
 register: datatype
-text: The data type for constraining a field value.
+text: "The form of the data in a specific field. This could be numerical ('2.50') or text ('abcdef')."
```

Comment:
I liked 'constraining', but we should remove the examples as they're better covered by the datatype register and the technical specification.